wsgidav/fs_dav_provider.py

```diff
@@ -104,8 +104,8 @@ def delete(self):
         if self.provider.readonly:
             raise DAVError(HTTP_FORBIDDEN)
         os.unlink(self._file_path)
-        self.remove_all_properties(recursive=True)
-        self.remove_all_locks(recursive=True)
+        self.remove_all_properties(recursive=False)
```

Comment:
Why was this behavior changed (from the docstring, the delete() method is supposed to be recursive)?

Reply:
If it's a resource why should locks be removed recursively? For a collection it seems logical, we can find locks below the directories, but not for a file. Right ?

Reply:
I understand, it's ok :)